docs/migrating/from-create-react-app.md

```diff
@@ -41,7 +41,7 @@ Here's an example `package.json`:
 
 Create React App uses the `public` directory for the [entry HTML file](https://create-react-app.dev/docs/using-the-public-folder), whereas Next.js uses it for static assets. It's possible to add static assets here, but Create React App recommends importing them directly from JavaScript files.
 
-- Move any images, fonts, or other static assets to `public`.
+- Move any images, fonts, or other static assets to `public`. In a Next.js application, this directory is kept at the root of your application.
```

Comment:
Maybe we can reword the above paragraph to mention that when migrating from CRA, the location of the `public` directory stays the same, at the projects root. And remove this addition:
> In a Next.js application, this directory is kept at the root of your application.